src/editortoolkit.cpp

```diff
@@ -417,6 +421,18 @@ bool EditorToolkit::Drag(std::string elementId, int x, int y, bool isChain)
 
         return true; // Can't reorder by layer since staves contain layers
     }
+    else if (element->Is(SYL)) {
+        Syl *syl = dynamic_cast<Syl *>(element);
+        if (!syl->HasFacs()) {
+            LogError("Syl (boundingbox) dragging is only supported for syls with facsimiles!");
+            return false;
+        }
+        FacsimileInterface *fi = (*syl).GetFacsimileInterface();
+        assert(fi);
+        if (fi->GetZone() != nullptr) {
```

Comment:
`nullptr` -> `NULL` to fit with the general style of Verovio, even if I do personally prefer `nullptr`.